packages/1155-contracts/src/minters/merkle/ZoraCreatorMerkleMinterStrategy.sol

```diff
@@ -58,12 +59,8 @@ contract ZoraCreatorMerkleMinterStrategy is Enjoy, SaleStrategy, LimitedMintPerA
 
     /// @notice Storage for allowed merkle settings for the sales configuration
     mapping(address => mapping(uint256 => MerkleSaleSettings)) public allowedMerkles;
-    // target -> tokenId -> settings
 
-    error SaleEnded();
-    error SaleHasNotStarted();
-    error WrongValueSent();
-    error InvalidMerkleProof(address mintTo, bytes32[] merkleProof, bytes32 merkleRoot);
+    // target -> tokenId -> settings
```

Comment:
do we still want to include this comment?

Reply:
no we should remove good catch